docs/getting-started/configure-integration-policy.asciidoc

```diff
@@ -42,6 +42,8 @@ Malware protection levels are:
 +
 TIP: Platinum and Enterprise customers can customize these notifications using the `Elastic Security {action} {filename}` syntax.
 
+Malware protection also allows you to implement a blocklist to prevent specified applications from running on hosts, extending the list of processes that {endpoint-sec} considers malicious. Use the **Blocklist enabled** toggle to enable or disable this feature for all hosts associated with the integration policy. To configure the blocklist, refer to <<blocklist>>.
```

Comment:
> Malware protection also allows you to implement a blocklist...

Maybe a better word here is "manage" in place of "implement" ?

suggestion:
> Malware protection also allows you to manage a blocklist...

Reply:
```suggestion
Malware protection also allows you to manage a blocklist to prevent specified applications from running on hosts, extending the list of processes that {endpoint-sec} considers malicious. Use the **Blocklist enabled** toggle to enable or disable this feature for all hosts associated with the integration policy. To configure the blocklist, refer to <<blocklist>>.
```